webpack.config.js

```diff
@@ -170,3 +172,21 @@ module.exports = {
     topLevelAwait: true,
   },
 };
+
+module.exports = (env, argv) => {
+  // since STB  isolates the build, we need to compile the translations here for releases
```

Comment:
```suggestion
  // Sphinx Theme Builder creates a completely isolated working directory
  // when packaging the theme. That means that we cannot follow a workflow
  // in which we first compile the translations and then build the theme. We
  // must compile the translations as part of the build process, which calls
  // `npm run-script build`, which in turn calls `webpack`.
```

Reply:
Shall we just link to the stb docs? All these steps are explained in https://sphinx-theme-builder.readthedocs.io/en/latest/build-process/